NEXT_CHANGELOG.md

```diff
@@ -9,11 +9,7 @@
 ### Dependency updates
 
 ### Bundles
-* Remove resources.apps.config section ([#3680](https://github.com/databricks/cli/pull/3680))
-* Prompt for serverless compute in `dbt-sql` template (defaults to `yes`) ([#3668](https://github.com/databricks/cli/pull/3668))
```

Comment:
Please check, I think you're accidentally removing several changes from `main` (both in the changelog and actual changes in for example `libs/template`)